packages/bpk-tokens/formatters/adjust-typography.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import androidTokenAliases from '../src/android/base/aliases.json';
+import iosTokenAliases from '../src/ios/base/aliases.json';
+
+const sfProTracking = {
+  '6': 41,
+  '8': 26,
+  '9': 19,
+  '10': 12,
+  '11': 6,
+  '12': 0,
+  '13': -6,
+  '14': -11,
+  '15': -16,
+  '16': -20,
+  '17': -24,
+  '18': -25,
+  '19': -26,
+  '20': 19,
+  '21': 17,
+  '22': 16,
+  '24': 15,
+  '25': 14,
+  '27': 13,
+  '30': 12,
+  '33': 11,
+  '36': 11,
```

Comment:
Where is this value from? It's not in the table on https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/